src/reminders/index.js

```diff
@@ -0,0 +1,43 @@
+import { NativeModules } from 'react-native';
```

Comment:
DRY !

Reply:
Fair - I guess this just becomes a readme change? I was wrapping the methods so things did not crash when run in a context other than and app with the Zapp SDK.

Reply:
Is it worth keeping the lint / test framework stuff?

Reply:
wrapping is fine, but you can actually wrap all the methods in one - it's the same syntax an all of them.

something like
```
export [...methods].reduce((_export, method) => {
  export[method] = args => { /* do your stuff */ }
}, {});
```

Reply:
tried to DRY it out as much as possible - let me know what you think